sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -398,15 +398,9 @@ trait CheckAnalysis extends PredicateHelper with LookupCatalog with QueryErrorsB
                   }
                 }
               case e: Attribute if groupingExprs.isEmpty =>
```

Comment:
Seems `e` is no longer used, replace with `_` ?

Reply:
replaced